README.md

```diff
@@ -1 +1,45 @@
-# first_capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone: setup and mobile version skeleton
+
+![screenshot](./Screenshot_2.png)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://laylooo.github.io/first_capstone/)
+
+## Getting Started
+
+Just simply clone or download the project and open index.html file with your browser.
+
+## Authors
+
+👤 **Laylo**
+
+- GitHub: [@githubhandle](https://github.com/Laylooo)
+- Twitter: [@twitterhandle](https://twitter.com/home?lang=en)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/laylo-khodjaeva-05a972207/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] OPTIONAL
It's a nice and smart thing to give appropriate credit to **Cindy Shin**, the author of the original design, as required by the Creative Commons License. Kindly ensure to add this under the "Acknowledgements" section of your ReadMe